src/obb_rules/serializer/reader.cljc

```diff
@@ -66,16 +67,19 @@
   (->> (string/split s (re-pattern common/action-separator))
        (mapv str->action)))
 
+(defn- gather-attributes
+  "Returns the given attribute bag with the raw attribute added"
```

Comment:
I think this docstring could be a bit more clear. I suggest something along this lines instead:

`"Parses string representation of an attribute and associates it to the given attrs map"`